src/components/tree.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState, useEffect } from "react";
+import {Tree} from '@react95/core';
+import { getTree} from '../Api';
+
+
+function TreeStruture() {
+    const [contentTree,setContentTree] = useState([]);
+    useEffect(() => {
+      const fetchTree = async () => {
+          const contentTree = await getTree();
+          setContentTree(contentTree);
+      } 
+      fetchTree();
+    },[]);
+
+
+  const label = function (str) {
+      return str.substring(str.lastIndexOf('/')+1);
+  }
```

Comment:
Move this function outside the React component (or to a separate utils file/folder) and rename it to `getLabel` or `generateLabel`. This will prevent the function to be recreated on every render.